### PR TITLE
feat(#arnjrd_#b0n2e4): add PayBill presentational component

### PR DESCRIPTION
**Pull request by dannylekim**

### CODE REVIEW

**Comment by dannylekim** [comment]
Task linked: [CU-arnjrd Pay bill](https://app.clickup.com/t/arnjrd)

**Comment by dannylekim** [comment]
Task linked: [CU-b0n2e4 Create presentational component](https://app.clickup.com/t/b0n2e4)

File: src/components/PayBill/Paybill.test.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import PayBill from "./PayBill";
+import React from "react";
+
+describe("PayBill", () => {
+  describe("render", () => {
+    describe("snapshots 📸", () => {
```
Comment (alphakennyn) [inline src/components/PayBill/Paybill.test.jsx:6]: maybe we can show different snaps when `billName` and `amountRemainingToPay` props are defined

Comment (dannylekim) [reply src/components/PayBill/Paybill.test.jsx:6]: Sure, I can add that in a bit

**Review by alphakennyn — APPROVED** [review]

**Pull request merged**